examples/rustapi_module/tests/test_datetime.py

```diff
@@ -248,6 +248,10 @@ def test_datetime_from_timestamp(dt):
     ts = get_timestamp(dt)
     assert rdt.datetime_from_timestamp(ts) == pdt.datetime.fromtimestamp(ts)
 
+def test_datetime_from_timestamp_with_falsifying_example():
```

Comment:
I don't think this needs to be a separate test, you can just use the `example` decorator with the existing test:

```python
@given(...)
@example(dt=datetime(1970, 1, 1, 0, 0))
def datetime_from_timestamp(dt):
    ...
```

Reply:
Sure, that's nicer. Do you have any idea what the problem could be?

Reply:
Hard to tell without a Windows computer. Timestamp support is OS-defined, so I am not terribly surprised that Windows is failing but not Linux. It does seem weird that it's failing with Python 3.6 and not 3.5, and that it only just started failing (I would be pretty surprised if `1970-01-01T00:00:00` is an unusual value for `hypothesis` to pass for datetimes).

Reply:
Not sure when it started failing, but this one is from two months ago: https://ci.appveyor.com/project/fafhrd91/pyo3/builds/24579560/job/b6eddxm0jdjgl7qs